src/TrackerCouncil.Smz3.UI/Views/MultiplayerStatusWindow.axaml.cs

```diff
@@ -97,7 +97,7 @@ private void OpenFolderMenuItem_OnClick(object? sender, RoutedEventArgs e)
 
     private void OpenTrackerMenuItem_OnClick(object? sender, RoutedEventArgs e)
     {
-        _service?.LaunchTracker();
+        _ = _service?.LaunchTracker();
```

Comment:
I know it really doesn't matter, but C# allows for `async void` specifically for event handlers so you can do this with `await`.

Reply:
I've been doing that more, I think I just forgot here (or maybe I did it because _technically_ the service can be null, so I'd have to make sure it's not null since I don't think C# likes nullable awaits).

That being said, I'm not sure if this is Rider specific or also a thing in Visual Studio, but it's started wanting try catches around the entire code in `async void` functions because unhandled exceptions can cause unexpected behaviors. (Well, unexpected in that `async Task` methods will typically silently fail on unhandled exceptions whereas `async void` methods will kill the entire application.)